python/ray/rllib/agents/ppo/ppo.py

```diff
@@ -150,6 +150,8 @@ def update(pi, pi_id):
         return res
 
     def _validate_config(self):
+        if self.config["entropy_coeff"] < 0:
```

Comment:
```suggestion
        if self.config["entropy_coeff"] <= 0:
```
Or alternatively change the error blow to "must be non-negative".

Reply:
Fixed